lib/import/admin.js

```diff
@@ -142,17 +150,57 @@ function save_features(feature_collection) {
 }
 
 /**
+ * Get hash of admin to population
+ * mutates `feature_collection`.
+ *
+ * @param{string} country_code - Country Code
+ * @param{array} feature_collection - Feature collection
+ * @return{Promise} Fulfilled when csv is read.
+ */
+function world_pop(country_code, feature_collection) {
+  var path = './data/population/';
+  var file = 'worldpop_' + country_code + '.csv';
```

Comment:
Please change `world_pop` to take the full file path as an argument, based on a new command line flag.

I know it may seem silly to require this on the command line when it's always the same, but it can our lives a lot simpler in the future if we need to call this from other code/docker setup scripts/cron jobs/etc.

In a follow-on CL, we could look at setting up defaults for these file paths based on the country code to save us from explicitly typing them out, but still allow full flexibility when we need it.